src/api/event/content-types/event/schema.json

```diff
@@ -34,6 +32,18 @@
       "type": "relation",
       "relation": "oneToOne",
       "target": "plugin::users-permissions.user"
+    },
+    "action": {
```

Comment:
I think we should consider `entityType` + `action` when we define the enum values. What does `entityType` Comment with `action` Idea Created mean? It probably doesn't make sense, but it's a valid combination.
I don't have a better suggestion right now. Each `entityType` has its set of actions.